spec/use_cases/source_tag_spec.rb

```diff
@@ -9,7 +9,16 @@
 
   let(:ref) { %w[sun mon tue wed thu fri sat] }
 
-  it 'check tag each day of the week' do
-    (0..6).map { |n| expect(tags[n]).to eq("feed_#{ref[n]}") }
+  it 'include tag each day of the week' do
+    (0..6).map { |n| expect(tags[n]).to include("feed_#{ref[n]}") }
+  end
+
+  it 'include tag #feed_wday on weekdays' do
+    (1..5).map { |n| expect(tags[n]).to include("feed_wday") }
+  end
+
+  it 'do not include tag #feed_wday on holiday' do
+    expect(tags[0]).not_to include("feed_wday")
+    expect(tags[6]).not_to include("feed_wday")
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->